### PR TITLE
LPS-78882 Detach window resize listener when destroying the CKEditor …

### PR DESCRIPTION
**Pull request by gregory-bretall**
…Instance

Changed the fix based on @jbalsas suggestions here: https://github.com/jbalsas/liferay-portal/pull/1094#issuecomment-374593465

### CODE REVIEW

**Comment by gregory-bretall** [comment]
ci:test

**Comment by jbalsas** [comment]
Hey @gregory-bretall, you should just push the handle into the `eventHandles` array and let the existing code do the `detach`.

**Comment by gregory-bretall** [comment]
Hi @jbalsas and @jonmak08 I made and tested the suggested change and pushed that version to this branch. Please let me know if you see any other issues with my proposed solution.

File: modules/apps/foundation/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor.jsp [hunk 1]
```diff
@@ -520,7 +520,7 @@ name = HtmlUtil.escapeJS(name);
 				<c:if test="<%= !(inlineEdit && Validator.isNotNull(inlineEditSaveURL)) %>">
 					var initialEditor = CKEDITOR.instances['<%= name %>'].id;
 
-					A.getWin().on(
+					eventHandles.push(A.getWin().on(
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor.jsp]: SF -> this should be on a different line 😉

File: modules/apps/foundation/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor.jsp [hunk 2]
```diff
@@ -552,7 +552,7 @@ name = HtmlUtil.escapeJS(name);
 							},
 							250
 						)
-					);
+					));
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor.jsp]: SF -> following the above comment, we will need another line here as well, and an indentation for the code between 😉

**Comment by jbalsas** [comment]
Looks good, @gregory-bretall, just SF it and LGTM 👍

**Comment by jonmak08** [comment]
Reviewed. Thanks!
Pull request submitted at: https://github.com/jbalsas/liferay-portal/pull/1097

View just my changes: https://github.com/jonmak08/liferay-portal/compare/gregory-bretall:cf935aad09...1a190b9712
View total diff: https://github.com/jonmak08/liferay-portal/compare/53eedc4c48...1a190b9712

**Pull request closed without merging**